toolbox/githubctl/main.go

```diff
@@ -32,7 +32,8 @@ import (
 
 var (
 	owner                 = flag.String("owner", "istio", "Github owner or org")
-	tokenFile             = flag.String("token_file", "", "File containing Github API Access Token")
+	accessToken           = flag.String("access_token", "", "Github API Access Token")
```

Comment:
this does not seem very safe to pass it directly in the cli as it would appear in the logs.